Nikon-Patch/Nikon-Patch/D5300.cs

```diff
@@ -12,11 +12,52 @@ public D5300_0101()
 
     class D5300_0102 : Firmware
     {
+        //24/20=42/35mbps?
+        static byte[] mbps_24_20be = { 0xC0, 0x5D, 0x00, 0x00, 0x20, 0x4E }; 
+        static byte[] mbps_64_60be = { 0x00, 0xFA, 0x00, 0x00, 0x60, 0xEA }; 
+        
+        //the above value divided by 1,000?
```

Comment:
this thousands smaller where commented as such because these 2 (now 3) line where below, so the order of the 3 set of rates would need to be swapped to make sense. The also the last 2 characters where `be` for everything as it stood for "bitrate encoding", but these three had a t because they were "thousands" possibly should have been a `k` for kbps.